### ISSUE TITLE
Remove priority when task marked as complete

### ISSUE THREAD

**Issue by anthonyesau** [davraamides/todotxt-mode#20]
Much appreciation for this extension. Thanks for the time and effort you've put into it!

An item for consideration: Would it be possible to remove the priority of tasks when they are marked complete using the _Toggle Task Completion_ command? Swiftodo, another todo.txt manager, doesn't recognize tasks as complete when the priority follows the initial x. The spec at [todotxt/todo.txt: ‼️ A complete primer on the whys and hows of todo.txt.](https://github.com/todotxt/todo.txt) also says that "The date of completion appears directly after the x, separated by a space." So it sounds like removing the priority would be the more standard behavior. 

An example
```
Original incomplete task
(A) 2020-09-29 Reply to email from Jim

Current result from toggling task completion
x (A) 2020-10-01 2020-09-29 Reply to email from Jim

Desired result
x 2020-10-01 2020-09-29 Reply to email from Jim
```

**Comment by davraamides** [comment]
I think that makes sense. I have a few queued up changes to the extension
that I need to knock out so I can look to include this in that update. The
one side-effect of such a change is that if you "toggle" a prioritized task
to completed and then uncompleted, you'd lose the original priority
(without preserving it somehow such as tacking on a pri:A tag which I find
a little messy).

**Comment by davraamides** [comment]
Implemented in release 1.4.17
